appdaemontestframework/init_framework.py

```diff
@@ -79,3 +84,10 @@ def log_log(msg, level='INFO'):
 
     hass_functions['error'].side_effect = log_error
     hass_functions['log'].side_effect = log_log
+
+def _mock_hass_init(hass_functions):
+    """Mock the Hass object init and set up class attributes that are used by automations"""
+    def hass_init_mock(self, ad, name, logger, error, args, config, app_config, global_vars):
```

Comment:
Please add a `_` to all the parameters we don't use.
So:
```python
def hass_init_mock(self, _ad, name, _logger, _error, _args, _config, _app_config, _global_vars):

```

It's a naming convention in Python to clearly express which parameter is used and which won't be touched in the execution of the function :)

Reply:
:+1: all set.